src/nodes.coffee

```diff
@@ -9,6 +9,12 @@ Error.stackTraceLimit = Infinity
 {RESERVED, STRICT_PROSCRIBED} = require './lexer'
 iced = require 'iced-runtime'
 
+# At this stage of development of this branch, it's easy to mess up
+# runtimes. The one in node_modules/, which is used by default, will
+# probably be incorrect.
+if not iced.const.iterator
+  throw new Error 'Wrong iced runtime!'
```

Comment:
I was thinking we should move the new runtime to a new npm module, what do you think? `iced-runtime-3` and emit code accordingly?  Alternatively, we can have the old runtime and the runtime coexist side-by-side in `iced-runtime` but we can use namespacing to separate them.  I'm indifferent between the two approaches but agree we need something, since we can't expect everyone to upgrade all at once.